package.json

```diff
@@ -73,5 +73,10 @@
   "lint-staged": {
     "*.{js,css,md,ts,json,yml,yaml}": "prettier --write"
   },
-  "packageManager": "yarn@3.2.2"
+  "packageManager": "yarn@3.2.3",
+  "resolutions": {
+    "ethereum-cryptography@1.1.2": "patch:ethereum-cryptography@npm%3A1.1.2#./.yarn/patches/ethereum-cryptography-npm-1.1.2-c16cfd7e8a.patch",
+    "ethereum-cryptography@^1.0.3": "patch:ethereum-cryptography@npm%3A1.1.2#./.yarn/patches/ethereum-cryptography-npm-1.1.2-c16cfd7e8a.patch",
+    "ethereum-cryptography@^0.1.3": "patch:ethereum-cryptography@npm%3A1.1.2#./.yarn/patches/ethereum-cryptography-npm-1.1.2-c16cfd7e8a.patch"
```

Comment:
@cds95 we should investigate if we can remove `0.1.3` version altogether and move to v1